README.md

```diff
@@ -46,7 +46,7 @@ custom:
     - bucketName: my-other-site
       localDir: path/to/other-site
       deleteRemoved: true # optional, indicates whether sync deletes files no longer present in localDir. Defaults to 'true'
-      acl: public-read # optional
```

Comment:
Change here is removing some non breaking spaces someone had introduced to the README

![image](https://github.com/k1LoW/serverless-s3-sync/assets/3787406/60475e14-6f16-496e-810a-24f8551f01e6)